setup.cfg

```diff
@@ -9,7 +9,7 @@ license = BSD 2-Clause
 url = https://sunpy.org
 edit_on_github = True
 github_project = sunpy/sunpy
-version = 1.0.dev
+version = 1.0
```

Comment:
Delete this completely

Reply:
It has to be there in some form otherwise it seems to error.

Reply:
It really shouldn't:

> Important: Ensure neither the [metadata] version option nor the [egg_info] section are defined, as these will interfere with setuptools_scm.

https://github.com/pypa/setuptools_scm/#setupcfg